unit/Makefile

```diff
@@ -106,8 +77,7 @@ BMC_DEPS =../src/cbmc/all_properties$(OBJEXT) \
           ../src/jsil/jsil_y.tab$(OBJEXT) \
           # Empty last line
 #
-CPROVER_LIBS =../src/java_bytecode/java_bytecode$(LIBEXT) \
-              ../src/miniz/miniz$(OBJEXT) \
+CPROVER_LIBS =../src/miniz/miniz$(OBJEXT) \
```

Comment:
miniz is being removed as well.

Reply:
I believe that is because the only use-case for miniz is for reading .jar files?

Reply:
Yes, miniz will be removed too.